static/src/stylesheets/module/content/_recipe-article.scss

```diff
@@ -0,0 +1,439 @@
+
+$grey:#dbdbdb;
+$grey-two:#f6f6f6;
+$grey-three:#767676;
+$purple:#B72166;
```

Comment:
I think all of these, except the purple are already defined in our colour palette. http://guardian.github.io/pasteup-palette/demo/. Please use this variables instead